README.md

```diff
@@ -115,6 +115,16 @@ This sample command provides the minimum mandatory arguments required to validat
 
 Currently only SPDX2.2 is supported.
 
+### SBOM Redact
+
+Use the tool to redact any references to files from a given SBOM or set of SBOMs with the command:
+
+```
+sbom-tool redact -sd <directory containing SBOMs to redact> -o <output path>
```

Comment:
I'm guessing that `-sp` and `-sd` are mutually exclusive. As such, would it make sense to document both options here?